dvconfig.py.sample

```diff
@@ -1,25 +1,35 @@
-base_url = 'http://localhost:8080'
-api_token = ''
+import os
+
+# Create the base_url from different parts (very usefull on K8s) or just
+# read completely from a single env var. Default to "http://localhost:8080"
+host = os.getenv('DATAVERSE_SERVICE_HOST', 'localhost')
+port = os.getenv('DATAVERSE_SERVICE_PORT_HTTP', '8080')
+proto = os.getenv('DATAVERSE_SERVICE_PORT_PROTO', 'http')
+subpath = os.getenv('DATAVERSE_SERVICE_SUBPATH', '')
+base_url = os.getenv('BASE_URL', proto+'://'+host+':'+port+subpath)
+
+api_token = os.getenv('API_TOKEN', '')
```

Comment:
I'm concerned that the change to `api_token` will break this:

https://github.com/IQSS/dataverse-ansible/blob/1449311f2bb5a260b08c001ab51f85c762b5a510/tasks/sampledata.yml#L52-L56

Reply:
Yes it will! But Ansible allows for a regex, so maybe we can change that one?